client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -318,11 +134,8 @@ export class DomainDetailsForm extends PureComponent {
 		return cartItems.hasOnlyDomainRegistrationsWithPrivacySupport( this.props.cart );
 	}
 
-	allDomainItemsHavePrivacy() {
-		return (
-			cartItems.getDomainRegistrationsWithoutPrivacy( this.props.cart ).length === 0 &&
-			cartItems.getDomainTransfersWithoutPrivacy( this.props.cart ).length === 0
-		);
+	allDomainRegistrationsHavePrivacy() {
```

Comment:
Why only domain registrations? Domain transfers are domains too! (or was this discussed with @aidvu?)

Reply:
This is probably why it needs a rebase. :)

Reply:
Aaaah, damn GitHub, confusing me! 🤜 :octocat:  🤛

Reply:
Going to rebase as the first task 👍